fastlane/Fastfile

```diff
@@ -269,12 +271,26 @@ platform :ios do
         UI.message "🤖 Testing schemas: #{schemas.join(', ')}"
         
         testplan = options[:testplan] || "AllTests"
+
+        failed_schemas = []
+
         schemas.each do |scheme|
             UI.message "🤖 Testing scheme #{scheme} with plan #{testplan}"
-            test_plan(build, scheme, testplan)
+            begin
+                test_plan(build, scheme, testplan)
+            rescue => e
+                failed_schemas << scheme
+                UI.error("Failed to test #{scheme}: #{e.message}")
+            end            
+        end
+                    
+        if failed_targets.any?
+            UI.important("The following schemas failed: #{failed_targets.join(', ')}")
+        else
+            UI.success("All tests passed successfully!")
         end
```

Comment:
this will run the next schema if the current one fails: not sure yet if that's a good idea. 

i.e.: WireFoundation fails (either build or tests) then WireDataModel (which depends on Foundation) will still run